### PR TITLE
[US393567] Update artifactory from devcloud to build.ge

### PR DESCRIPTION
**Pull request by emarmol**
[US393567](https://rally1.rallydev.com/#/detail/userstory/361940107604?fdp=true): Migrate UAA/UAA Dashboard from DevCloud Artifactory to new Build.GE Artifactory

Passing Build: https://i.ci.build.ge.com/bjmjek1k/ci/job/UAA/job/Build%20n%20Test/job/update-artifactory/15/

Uploaded Artifact: https://artifactory.build.ge.com/webapp/#/artifacts/browse/tree/General/MAAXA/builds/uaa/4.30.0-test

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -71,14 +71,7 @@ allprojects {
...
             }
-            url 'https://devcloud.swcoe.ge.com/artifactory/PREDIX'
-        }
-        maven {
-            credentials {
-              username geArtifactoryReader
-              password geArtifactoryReaderPassword
-            }
-            url 'https://devcloud.swcoe.ge.com/artifactory/PREDIX-SNAPSHOT'
+            url 'https://artifactory.build.ge.com/predix-virtual'
```
Comment (metacubed) [inline build.gradle]: This probably needs to be `MAAXA` and `MAAXA-SNAPSHOT`, not `predix-virtual`. It's used for resolving and deploying the top level gradle project.

File: build.gradle [hunk 2]
```diff
@@ -187,15 +180,7 @@ subprojects {
...
-            url 'https://devcloud.swcoe.ge.com/artifactory/PREDIX'
-        }
-
-        maven {
-            credentials {
-                username geArtifactoryReader
-                password geArtifactoryReaderPassword
-            }
-            url 'https://devcloud.swcoe.ge.com/artifactory/PREDIX-SNAPSHOT'
+            url 'https://artifactory.build.ge.com/predix-virtual'
```
Comment (metacubed) [inline build.gradle]: This probably needs to be `MAAXA` and `MAAXA-SNAPSHOT`, not `predix-virtual`. It's used for resolving and deploying the child gradle projects (the actual functional ones).

File: build.gradle (hunk 1 above)
Comment (emarmol) [reply build.gradle]: `MAAXA` and `MAAXA-SNAPSHOT` weren't required here to deploy. We need predix-virtual for audit sdk dependency.

File: build.gradle (hunk 2 above)
Comment (emarmol) [reply build.gradle]: `MAAXA` and `MAAXA-SNAPSHOT` weren't required here to deploy. We need predix-virtual for audit sdk dependency.

**Pull request merged**